### ISSUE TITLE
Using "fill" on eraser

### ISSUE THREAD

**Issue by MithicSpirit** [xournalpp/xournalpp#3370]
**Is your feature request related to a problem? Please describe.**
I frequently need to erase large regions of strokes and doing so with the erase is somewhat annoying to get every little stroke, even with "delete stroke" enabled. Using select region and then deleting that region works but feels a bit jank.

**Describe the solution you'd like**
Adding support for the fill option for the eraser would fix this as I could just circle the region and everything inside would be erased.

**Comment by Technius** [comment]
Very interesting idea! I think this would be consistent with how the stroke tools are filled.

**Comment by K4LCIFER** [comment]
I agree with the jankiness of the current situation of selecting and deleting. That little red x that you have to click is a little tedious to hit at times. It'd be fantastic to be able to just select and delete in one fell swoop. Should this be part of the selection tool or the eraser tool, I wonder?

**Comment by ShaanMakoon** [comment]
I know that it has been a while, but I am new to the project, and would like to work on this issue.
